osu.Game/Beatmaps/Drawables/DifficultyIconTooltip.cs

```diff
@@ -146,7 +146,10 @@ public void SetContent(DifficultyIconTooltipContent content)
             approachRate.Text = @" AR: " + adjustedDifficulty.ApproachRate.ToString(@"0.##");
             overallDifficulty.Text = @" OD: " + adjustedDifficulty.OverallDifficulty.ToString(@"0.##");
 
-            length.Text = "Length: " + TimeSpan.FromMilliseconds(displayedContent.BeatmapInfo.Length / rate).ToString(@"mm\:ss");
+            TimeSpan lengthTimeSpan = TimeSpan.FromMilliseconds(displayedContent.BeatmapInfo.Length / rate);
+            length.Text = "Length: " + (lengthTimeSpan.Hours > 0
```

Comment:
This is not correct. Counterexample: try any timespan which is a multiple of 24 hours. You want

```suggestion
            length.Text = "Length: " + (lengthTimeSpan.TotalHours > 0
```

See [docs](https://learn.microsoft.com/en-us/dotnet/api/system.timespan.totalhours?view=net-8.0#remarks) to understand the difference.